### PR TITLE
DOC: Fix SA04 errors in docstrings xref #28792

### LINKED ISSUE #28792: DOC: Fix SA04 errors in docstrings
There are issues in some docstrings where the See Also section is not complete. All of the following errors exist because there is no description for a referenced function or method. If these can get fixed then CI will be able to check all of the See Also section errors! The command to check for the errors is `./scripts/validate_docstrings.py --errors=SA04`

```
pandas.melt: Missing description for See Also "DataFrame.melt" reference
pandas.melt: Missing description for See Also "pivot_table" reference
pandas.melt: Missing description for See Also "DataFrame.pivot" reference
pandas.melt: Missing description for See Also "Series.explode" reference
pandas.merge_ordered: Missing description for See Also "merge" reference
pandas.merge_ordered: Missing description for See Also "merge_asof" reference
pandas.merge_asof: Missing description for See Also "merge" reference
pandas.merge_asof: Missing description for See Also "merge_ordered" reference
pandas.unique: Missing description for See Also "Index.unique" reference
pandas.unique: Missing description for See Also "Series.unique" reference
pandas.eval: Missing description for See Also "DataFrame.query" reference
pandas.eval: Missing description for See Also "DataFrame.eval" reference
pandas.api.extensions.register_dataframe_accessor: Missing description for See Also "register_series_accessor" reference
pandas.api.extensions.register_dataframe_accessor: Missing description for See Also "register_index_accessor" reference
pandas.api.extensions.register_series_accessor: Missing description for See Also "register_dataframe_accessor" reference
pandas.api.extensions.register_series_accessor: Missing description for See Also "register_index_accessor" reference
pandas.api.extensions.register_index_accessor: Missing description for See Also "register_dataframe_accessor" reference
pandas.api.extensions.register_index_accessor: Missing description for See Also "register_series_accessor" reference
pandas.api.extensions.ExtensionDtype: Missing description for See Also "extensions.register_extension_dtype" reference
pandas.api.extensions.ExtensionDtype: Missing description for See Also "extensions.ExtensionArray" reference
pandas.api.extensions.ExtensionArray._from_factorized: Missing description for See Also "factorize" reference
pandas.api.extensions.ExtensionArray._from_factorized: Missing description for See Also "ExtensionArray.factorize" reference
pandas.api.extensions.ExtensionArray._values_for_argsort: Missing description for See Also "ExtensionArray.argsort" reference
pandas.api.extensions.ExtensionArray.take: Missing description for See Also "numpy.take" reference
pandas.api.extensions.ExtensionArray.take: Missing description for See Also "api.extensions.take" reference
pandas.io.formats.style.Styler: Missing description for See Also "DataFrame.style" reference
pandas.io.formats.style.Styler.applymap: Missing description for See Also "Styler.where" reference
pandas.io.formats.style.Styler.where: Missing description for See Also "Styler.applymap" reference
pandas.io.formats.style.Styler.export: Missing description for See Also "Styler.use" reference
pandas.io.formats.style.Styler.use: Missing description for See Also "Styler.export" reference
pandas.DataFrame.shape: Missing description for See Also "ndarray.shape" reference
pandas.DataFrame.empty: Missing description for See Also "Series.dropna" reference
pandas.DataFrame.empty: Missing description for See Also "DataFrame.dropna" reference
pandas.DataFrame.pipe: Missing description for See Also "DataFrame.apply" reference
pandas.DataFrame.pipe: Missing description for See Also "DataFrame.applymap" reference
pandas.DataFrame.pipe: Missing description for See Also "Series.map" reference
pandas.DataFrame.corr: Missing description for See Also "DataFrame.corrwith" reference
pandas.DataFrame.corr: Missing description for See Also "Series.corr" reference
pandas.DataFrame.corrwith: Missing description for See Also "DataFrame.corr" reference
pandas.DataFrame.filter: Missing description for See Also "DataFrame.loc" reference
pandas.DataFrame.idxmax: Missing description for See Also "Series.idxmax" reference
pandas.DataFrame.idxmin: Missing description for See Also "Series.idxmin" reference
pandas.DataFrame.melt: Missing description for See Also "melt" reference
pandas.DataFrame.melt: Missing description for See Also "pivot_table" reference
pandas.DataFrame.melt: Missing description for See Also "DataFrame.pivot" reference
pandas.DataFrame.melt: Missing description for See Also "Series.explode" reference
pandas.DataFrame.asfreq: Missing description for See Also "reindex" reference
pandas.DataFrame.to_json: Missing description for See Also "read_json" reference
pandas.DataFrame.style: Missing description for See Also "io.formats.style.Styler" reference
pandas.Series.put: Missing description for See Also "numpy.ndarray.put" reference
pandas.Series.to_list: Missing description for See Also "numpy.ndarray.tolist" reference
pandas.Series.add: Missing description for See Also "Series.radd" reference
pandas.Series.sub: Missing description for See Also "Series.rsub" reference
pandas.Series.mul: Missing description for See Also "Series.rmul" reference
pandas.Series.div: Missing description for See Also "Series.rtruediv" reference
pandas.Series.truediv: Missing description for See Also "Series.rtruediv" reference
pandas.Series.floordiv: Missing description for See Also "Series.rfloordiv" reference
pandas.Series.mod: Missing description for See Also "Series.rmod" reference
pandas.Series.pow: Missing description for See Also "Series.rpow" reference
pandas.Series.radd: Missing description for See Also "Series.add" reference
pandas.Series.rsub: Missing description for See Also "Series.sub" reference
pandas.Series.rmul: Missing description for See Also "Series.mul" reference
pandas.Series.rdiv: Missing description for See Also "Series.truediv" reference
pandas.Series.rtruediv: Missing description for See Also "Series.truediv" reference
pandas.Series.rfloordiv: Missing description for See Also "Series.floordiv" reference
pandas.Series.rmod: Missing description for See Also "Series.mod" reference
pandas.Series.rpow: Missing description for See Also "Series.pow" reference
pandas.Series.lt: Missing description for See Also "Series.None" reference
pandas.Series.gt: Missing description for See Also "Series.None" reference
pandas.Series.le: Missing description for See Also "Series.None" reference
pandas.Series.ge: Missing description for See Also "Series.None" reference
pandas.Series.ne: Missing description for See Also "Series.None" reference
pandas.Series.eq: Missing description for See Also "Series.None" reference
pandas.Series.pipe: Missing description for See Also "DataFrame.apply" reference
pandas.Series.pipe: Missing description for See Also "DataFrame.applymap" reference
pandas.Series.pipe: Missing description for See Also "Series.map" reference
pandas.Series.quantile: Missing description for See Also "core.window.Rolling.quantile" reference
pandas.Series.quantile: Missing description for See Also "numpy.percentile" reference
pandas.Series.filter: Missing description for See Also "DataFrame.loc" reference
pandas.Series.argsort: Missing description for See Also "numpy.ndarray.argsort" reference
pandas.Series.searchsorted: Missing description for See Also "numpy.searchsorted" reference
pandas.Series.ravel: Missing description for See Also "numpy.ndarray.ravel" reference
pandas.Series.asfreq: Missing description for See Also "reindex" reference
pandas.Series.dt.to_pytimedelta: Missing description for See Also "datetime.timedelta" reference
pandas.Series.cat.categories: Missing description for See Also "rename_categories" reference
pandas.Series.cat.categories: Missing description for See Also "reorder_categories" reference
pandas.Series.cat.categories: Missing description for See Also "add_categories" reference
pandas.Series.cat.categories: Missing description for See Also "remove_categories" reference
pandas.Series.cat.categories: Missing description for See Also "remove_unused_categories" reference
pandas.Series.cat.categories: Missing description for See Also "set_categories" reference
pandas.Series.cat.rename_categories: Missing description for See Also "reorder_categories" reference
pandas.Series.cat.rename_categories: Missing description for See Also "add_categories" reference
pandas.Series.cat.rename_categories: Missing description for See Also "remove_categories" reference
pandas.Series.cat.rename_categories: Missing description for See Also "remove_unused_categories" reference
pandas.Series.cat.rename_categories: Missing description for See Also "set_categories" reference
pandas.Series.cat.reorder_categories: Missing description for See Also "rename_categories" reference
pandas.Series.cat.reorder_categories: Missing description for See Also "add_categories" reference
pandas.Series.cat.reorder_categories: Missing description for See Also "remove_categories" reference
pandas.Series.cat.reorder_categories: Missing description for See Also "remove_unused_categories" reference
pandas.Series.cat.reorder_categories: Missing description for See Also "set_categories" reference
pandas.Series.cat.add_categories: Missing description for See Also "rename_categories" reference
pandas.Series.cat.add_categories: Missing description for See Also "reorder_categories" reference
pandas.Series.cat.add_categories: Missing description for See Also "remove_categories" reference
pandas.Series.cat.add_categories: Missing description for See Also "remove_unused_categories" reference
pandas.Series.cat.add_categories: Missing description for See Also "set_categories" reference
pandas.Series.cat.remove_categories: Missing description for See Also "rename_categories" reference
pandas.Series.cat.remove_categories: Missing description for See Also "reorder_categories" reference
pandas.Series.cat.remove_categories: Missing description for See Also "add_categories" reference
pandas.Series.cat.remove_categories: Missing description for See Also "remove_unused_categories" reference
pandas.Series.cat.remove_categories: Missing description for See Also "set_categories" reference
pandas.Series.cat.remove_unused_categories: Missing description for See Also "rename_categories" reference
pandas.Series.cat.remove_unused_categories: Missing description for See Also "reorder_categories" reference
pandas.Series.cat.remove_unused_categories: Missing description for See Also "add_categories" reference
pandas.Series.cat.remove_unused_categories: Missing description for See Also "remove_categories" reference
pandas.Series.cat.remove_unused_categories: Missing description for See Also "set_categories" reference
pandas.Series.cat.set_categories: Missing description for See Also "rename_categories" reference
pandas.Series.cat.set_categories: Missing description for See Also "reorder_categories" reference
pandas.Series.cat.set_categories: Missing description for See Also "add_categories" reference
pandas.Series.cat.set_categories: Missing description for See Also "remove_categories" reference
pandas.Series.cat.set_categories: Missing description for See Also "remove_unused_categories" reference
pandas.Series.to_json: Missing description for See Also "read_json" reference
pandas.Index: Missing description for See Also "DatetimeIndex" reference
pandas.Index: Missing description for See Also "TimedeltaIndex" reference
pandas.Index: Missing description for See Also "PeriodIndex" reference
pandas.Index: Missing description for See Also "Int64Index" reference
pandas.Index: Missing description for See Also "UInt64Index" reference
pandas.Index: Missing description for See Also "Float64Index" reference
pandas.Index.memory_usage: Missing description for See Also "numpy.ndarray.nbytes" reference
pandas.Index.argmin: Missing description for See Also "numpy.ndarray.argmin" reference
pandas.Index.argmax: Missing description for See Also "numpy.ndarray.argmax" reference
pandas.Index.take: Missing description for See Also "numpy.ndarray.take" reference
pandas.Index.putmask: Missing description for See Also "numpy.ndarray.putmask" reference
pandas.Index.unique: Missing description for See Also "unique" reference
pandas.Index.unique: Missing description for See Also "Series.unique" reference
pandas.Index.ravel: Missing description for See Also "numpy.ndarray.ravel" reference
pandas.Index.to_list: Missing description for See Also "numpy.ndarray.tolist" reference
pandas.Index.searchsorted: Missing description for See Also "numpy.searchsorted" reference
pandas.CategoricalIndex.rename_categories: Missing description for See Also "reorder_categories" reference
pandas.CategoricalIndex.rename_categories: Missing description for See Also "add_categories" reference
pandas.CategoricalIndex.rename_categories: Missing description for See Also "remove_categories" reference
pandas.CategoricalIndex.rename_categories: Missing description for See Also "remove_unused_categories" reference
pandas.CategoricalIndex.rename_categories: Missing description for See Also "set_categories" reference
pandas.CategoricalIndex.reorder_categories: Missing description for See Also "rename_categories" reference
pandas.CategoricalIndex.reorder_categories: Missing description for See Also "add_categories" reference
pandas.CategoricalIndex.reorder_categories: Missing description for See Also "remove_categories" reference
pandas.CategoricalIndex.reorder_categories: Missing description for See Also "remove_unused_categories" reference
pandas.CategoricalIndex.reorder_categories: Missing description for See Also "set_categories" reference
pandas.CategoricalIndex.add_categories: Missing description for See Also "rename_categories" reference
pandas.CategoricalIndex.add_categories: Missing description for See Also "reorder_categories" reference
pandas.CategoricalIndex.add_categories: Missing description for See Also "remove_categories" reference
pandas.CategoricalIndex.add_categories: Missing description for See Also "remove_unused_categories" reference
pandas.CategoricalIndex.add_categories: Missing description for See Also "set_categories" reference
pandas.CategoricalIndex.remove_categories: Missing description for See Also "rename_categories" reference
pandas.CategoricalIndex.remove_categories: Missing description for See Also "reorder_categories" reference
pandas.CategoricalIndex.remove_categories: Missing description for See Also "add_categories" reference
pandas.CategoricalIndex.remove_categories: Missing description for See Also "remove_unused_categories" reference
pandas.CategoricalIndex.remove_categories: Missing description for See Also "set_categories" reference
pandas.CategoricalIndex.remove_unused_categories: Missing description for See Also "rename_categories" reference
pandas.CategoricalIndex.remove_unused_categories: Missing description for See Also "reorder_categories" reference
pandas.CategoricalIndex.remove_unused_categories: Missing description for See Also "add_categories" reference
pandas.CategoricalIndex.remove_unused_categories: Missing description for See Also "remove_categories" reference
pandas.CategoricalIndex.remove_unused_categories: Missing description for See Also "set_categories" reference
pandas.CategoricalIndex.set_categories: Missing description for See Also "rename_categories" reference
pandas.CategoricalIndex.set_categories: Missing description for See Also "reorder_categories" reference
pandas.CategoricalIndex.set_categories: Missing description for See Also "add_categories" reference
pandas.CategoricalIndex.set_categories: Missing description for See Also "remove_categories" reference
pandas.CategoricalIndex.set_categories: Missing description for See Also "remove_unused_categories" reference
pandas.MultiIndex.to_frame: Missing description for See Also "DataFrame" reference
pandas.DatetimeIndex.indexer_at_time: Missing description for See Also "indexer_between_time" reference
pandas.DatetimeIndex.indexer_at_time: Missing description for See Also "DataFrame.at_time" reference
pandas.DatetimeIndex.indexer_between_time: Missing description for See Also "indexer_at_time" reference
pandas.DatetimeIndex.indexer_between_time: Missing description for See Also "DataFrame.between_time" reference
pandas.DatetimeIndex.mean: Missing description for See Also "numpy.ndarray.mean" reference
pandas.TimedeltaIndex.mean: Missing description for See Also "numpy.ndarray.mean" reference
pandas.core.window.rolling.Rolling.aggregate: Missing description for See Also "Series.rolling" reference
pandas.core.window.rolling.Rolling.aggregate: Missing description for See Also "DataFrame.rolling" reference
pandas.core.window.expanding.Expanding.aggregate: Missing description for See Also "DataFrame.expanding.aggregate" reference
pandas.core.window.expanding.Expanding.aggregate: Missing description for See Also "DataFrame.rolling.aggregate" reference
pandas.core.window.expanding.Expanding.aggregate: Missing description for See Also "DataFrame.aggregate" reference
pandas.plotting.deregister_matplotlib_converters: Missing description for See Also "register_matplotlib_converters" reference
pandas.plotting.register_matplotlib_converters: Missing description for See Also "deregister_matplotlib_converters" reference
pandas.read_html: Missing description for See Also "read_csv" reference
pandas.read_sql_query: Missing description for See Also "read_sql" reference
pandas.tseries.frequencies.to_offset: Missing description for See Also "DateOffset" reference
pandas.core.groupby.GroupBy.all: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.all: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.any: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.any: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.bfill: Missing description for See Also "Series.backfill" reference
pandas.core.groupby.GroupBy.bfill: Missing description for See Also "DataFrame.backfill" reference
pandas.core.groupby.GroupBy.bfill: Missing description for See Also "Series.fillna" reference
pandas.core.groupby.GroupBy.bfill: Missing description for See Also "DataFrame.fillna" reference
pandas.core.groupby.GroupBy.count: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.count: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.cummax: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.cummax: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.cummin: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.cummin: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.cumprod: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.cumprod: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.cumsum: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.cumsum: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.ffill: Missing description for See Also "Series.pad" reference
pandas.core.groupby.GroupBy.ffill: Missing description for See Also "DataFrame.pad" reference
pandas.core.groupby.GroupBy.ffill: Missing description for See Also "Series.fillna" reference
pandas.core.groupby.GroupBy.ffill: Missing description for See Also "DataFrame.fillna" reference
pandas.core.groupby.GroupBy.head: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.head: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.mean: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.mean: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.median: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.median: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.nth: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.nth: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.ohlc: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.ohlc: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.rank: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.rank: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.pct_change: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.pct_change: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.size: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.size: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.sem: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.sem: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.std: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.std: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.var: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.var: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.GroupBy.tail: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.GroupBy.tail: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.all: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.all: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.any: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.any: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.bfill: Missing description for See Also "Series.backfill" reference
pandas.core.groupby.DataFrameGroupBy.bfill: Missing description for See Also "DataFrame.backfill" reference
pandas.core.groupby.DataFrameGroupBy.bfill: Missing description for See Also "Series.fillna" reference
pandas.core.groupby.DataFrameGroupBy.bfill: Missing description for See Also "DataFrame.fillna" reference
pandas.core.groupby.DataFrameGroupBy.corr: Missing description for See Also "DataFrame.corrwith" reference
pandas.core.groupby.DataFrameGroupBy.corr: Missing description for See Also "Series.corr" reference
pandas.core.groupby.DataFrameGroupBy.cummax: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.cummax: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.cummin: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.cummin: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.cumprod: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.cumprod: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.cumsum: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.cumsum: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.ffill: Missing description for See Also "Series.pad" reference
pandas.core.groupby.DataFrameGroupBy.ffill: Missing description for See Also "DataFrame.pad" reference
pandas.core.groupby.DataFrameGroupBy.ffill: Missing description for See Also "Series.fillna" reference
pandas.core.groupby.DataFrameGroupBy.ffill: Missing description for See Also "DataFrame.fillna" reference
pandas.core.groupby.DataFrameGroupBy.idxmax: Missing description for See Also "Series.idxmax" reference
pandas.core.groupby.DataFrameGroupBy.idxmin: Missing description for See Also "Series.idxmin" reference
pandas.core.groupby.DataFrameGroupBy.pct_change: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.pct_change: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.rank: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.rank: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.shift: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.shift: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.size: Missing description for See Also "Series.groupby" reference
pandas.core.groupby.DataFrameGroupBy.size: Missing description for See Also "DataFrame.groupby" reference
pandas.core.groupby.DataFrameGroupBy.corrwith: Missing description for See Also "DataFrame.corr" reference
pandas.core.resample.Resampler.__iter__: Missing description for See Also "GroupBy.__iter__" reference
pandas.core.resample.Resampler.apply: Missing description for See Also "DataFrame.groupby.aggregate" reference
pandas.core.resample.Resampler.apply: Missing description for See Also "DataFrame.resample.transform" reference
pandas.core.resample.Resampler.apply: Missing description for See Also "DataFrame.aggregate" reference
pandas.core.resample.Resampler.aggregate: Missing description for See Also "DataFrame.groupby.aggregate" reference
pandas.core.resample.Resampler.aggregate: Missing description for See Also "DataFrame.resample.transform" reference
pandas.core.resample.Resampler.aggregate: Missing description for See Also "DataFrame.aggregate" reference
pandas.core.resample.Resampler.ffill: Missing description for See Also "Series.fillna" reference
pandas.core.resample.Resampler.ffill: Missing description for See Also "DataFrame.fillna" reference
pandas.core.resample.Resampler.pad: Missing description for See Also "Series.fillna" reference
pandas.core.resample.Resampler.pad: Missing description for See Also "DataFrame.fillna" reference
pandas.core.resample.Resampler.asfreq: Missing description for See Also "Series.asfreq" reference
pandas.core.resample.Resampler.asfreq: Missing description for See Also "DataFrame.asfreq" reference
pandas.core.resample.Resampler.count: Missing description for See Also "Series.groupby" reference
pandas.core.resample.Resampler.count: Missing description for See Also "DataFrame.groupby" reference
pandas.core.resample.Resampler.mean: Missing description for See Also "Series.groupby" reference
pandas.core.resample.Resampler.mean: Missing description for See Also "DataFrame.groupby" reference
pandas.core.resample.Resampler.median: Missing description for See Also "Series.groupby" reference
pandas.core.resample.Resampler.median: Missing description for See Also "DataFrame.groupby" reference
pandas.core.resample.Resampler.ohlc: Missing description for See Also "Series.groupby" reference
pandas.core.resample.Resampler.ohlc: Missing description for See Also "DataFrame.groupby" reference
pandas.core.resample.Resampler.size: Missing description for See Also "Series.groupby" reference
pandas.core.resample.Resampler.size: Missing description for See Also "DataFrame.groupby" reference
pandas.core.resample.Resampler.sem: Missing description for See Also "Series.groupby" reference
pandas.core.resample.Resampler.sem: Missing description for See Also "DataFrame.groupby" reference
pandas.core.resample.Resampler.quantile: Missing description for See Also "Series.quantile" reference
pandas.core.resample.Resampler.quantile: Missing description for See Also "DataFrame.quantile" reference
pandas.core.resample.Resampler.quantile: Missing description for See Also "DataFrameGroupBy.quantile" reference
pandas.Timedelta.isoformat: Missing description for See Also "Timestamp.isoformat" reference
pandas.CategoricalDtype: Missing description for See Also "Categorical" reference
pandas.Categorical.categories: Missing description for See Also "rename_categories" reference
pandas.Categorical.categories: Missing description for See Also "reorder_categories" reference
pandas.Categorical.categories: Missing description for See Also "add_categories" reference
pandas.Categorical.categories: Missing description for See Also "remove_categories" reference
pandas.Categorical.categories: Missing description for See Also "remove_unused_categories" reference
pandas.Categorical.categories: Missing description for See Also "set_categories" reference
pandas.Series.as_matrix: Missing description for See Also "DataFrame.values" reference
pandas.Series.compress: Missing description for See Also "numpy.ndarray.compress" reference
pandas.Series.divide: Missing description for See Also "Series.rtruediv" reference
pandas.Series.divmod: Missing description for See Also "Series.rdivmod" reference
pandas.Series.multiply: Missing description for See Also "Series.rmul" reference
pandas.Series.nonzero: Missing description for See Also "numpy.nonzero" reference
pandas.Series.rdivmod: Missing description for See Also "Series.divmod" reference
pandas.Series.subtract: Missing description for See Also "Series.rsub" reference
pandas.Series.tolist: Missing description for See Also "numpy.ndarray.tolist" reference
pandas.DataFrame.as_matrix: Missing description for See Also "DataFrame.values" reference
```

### PR DESCRIPTION
**Pull request by AdrianMastronardi**
- [x] xref #28792 
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -2128,8 +2131,8 @@ def quantile(self, q=0.5, interpolation="linear"):
 
         See Also
         --------
-        core.window.Rolling.quantile
-        numpy.percentile
+        core.window.Rolling.quantile : Calculate the %(name)s quantile.
```
Comment (MarcoGorelli) [inline pandas/core/series.py]: Does this get substituted? Could you try [buliding the docs](https://pandas.pydata.org/docs/development/contributing.html#building-the-documentation) and posting screenshots?

File: pandas/core/frame.py [hunk 2]
```diff
@@ -7900,7 +7901,8 @@ def idxmin(self, axis=0, skipna=True) -> Series:
 
         See Also
         --------
-        Series.idxmin
+        Series.idxmin : : Return index *label* of the first occurrence
```
Comment (MarcoGorelli) [inline pandas/core/frame.py]: Was the double-colon intentional?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Thanks @AdrianMastronardi

Would this close #28792? If not, prob best to put "xref #28792"

File: pandas/core/frame.py (hunk 2 above)
Comment (AdrianMastronardi) [reply pandas/core/frame.py]: It is a typo. I will fix.

File: pandas/core/series.py (hunk 1 above)
Comment (AdrianMastronardi) [reply pandas/core/series.py]: Marco, thanks for your feedback. No, it doesn't get substituted. How shall I handle it?
screenshot

Comment (AdrianMastronardi) [reply pandas/core/series.py]: Shouldn't in this case the text be: `core.window.Rolling.quantile : Calculate the rolling quantile.`?

File: pandas/core/algorithms.py [hunk 3]
```diff
@@ -313,8 +313,8 @@ def unique(values):
 
     See Also
     --------
-    Index.unique
-    Series.unique
+    Index.unique  : Return Index with unique values from an Index object.
```
Comment (datapythonista) [inline pandas/core/algorithms.py]:
```suggestion
    Index.unique : Return unique values from an Index.
```

File: pandas/core/frame.py [hunk 4]
```diff
@@ -7900,7 +7901,8 @@ def idxmin(self, axis=0, skipna=True) -> Series:
 
         See Also
         --------
-        Series.idxmin
+        Series.idxmin : Return index *label* of the first occurrence
+            of minimum of values.
```
Comment (datapythonista) [inline pandas/core/frame.py]: A bit difficult to read IMHO.  Do you think `Return index of the minimum element` is good enough? May be not so precise, but to let people know if they are interested or not in this page should probably be enough.

File: pandas/core/frame.py [hunk 5]
```diff
@@ -7938,7 +7940,8 @@ def idxmax(self, axis=0, skipna=True) -> Series:
 
         See Also
         --------
-        Series.idxmax
+        Series.idxmax : Return index *label* of the first occurrence
+            of maximum of values.
```
Comment (datapythonista) [inline pandas/core/frame.py]: same

File: pandas/core/series.py (hunk 1 above)
Comment (datapythonista) [reply pandas/core/series.py]: I think `Calculate the rolling quantile.` is ok. Not sure what you wanted to get from `%(name)s`, but we only use variables when we reuse the same doctring for different objects, and some parts change (which I don't think it applies here).

**Review by datapythonista — COMMENTED** [review]
Nice work, couple of minor comments, but looks good, thanks.

File: pandas/core/frame.py [hunk 6]
```diff
@@ -358,9 +358,9 @@ class DataFrame(NDFrame):
     --------
     DataFrame.from_records : Constructor from tuples, also record arrays.
     DataFrame.from_dict : From dicts of Series, arrays, or dicts.
-    read_csv
-    read_table
-    read_clipboard
+    read_csv : Read a comma-separated values (csv) file into DataFrame.
+    read_table : Read general delimited file into DataFrame.
+    read_clipboard : Read text from clipboard and pass to read_table.
```
Comment (simonjayhawkins) [inline pandas/core/frame.py]:
```suggestion
    read_clipboard : Read text from clipboard into DataFrame.
```

File: pandas/core/frame.py [hunk 7]
```diff
@@ -7292,8 +7292,9 @@ def corr(self, method="pearson", min_periods=1) -> "DataFrame":
 
         See Also
         --------
-        DataFrame.corrwith
-        Series.corr
+        DataFrame.corrwith : Compute pairwise correlation between rows or
```
Comment (simonjayhawkins) [inline pandas/core/frame.py]:
```suggestion
        DataFrame.corrwith : Compute pairwise correlation with another DataFrame or Series.
```

File: pandas/core/frame.py [hunk 8]
```diff
@@ -7292,8 +7292,9 @@ def corr(self, method="pearson", min_periods=1) -> "DataFrame":
...
+            columns of two DataFrame objects.
```
Comment (simonjayhawkins) [inline pandas/core/frame.py]: remove

**Review by simonjayhawkins — COMMENTED** [review]
Thanks @AdrianMastronardi generally lgtm. a couple of suggestions.

File: pandas/core/frame.py (hunk 8 above)
Comment (simonjayhawkins) [reply pandas/core/frame.py]: just L7296 if accept above suggestion.

Comment (AdrianMastronardi) [reply pandas/core/frame.py]: Thanks for you feedback. I just incorporated your suggestions.

File: pandas/core/frame.py [hunk 9]
```diff
@@ -8039,7 +8040,7 @@ def idxmax(self, axis=0, skipna=True) -> Series:
 
         See Also
         --------
-        Series.idxmax
+        Series.idxmax : Return index of the maximun element.
```
Comment (simonjayhawkins) [inline pandas/core/frame.py]:
```suggestion
        Series.idxmax : Return index of the maximum element.
```

**Review by simonjayhawkins — CHANGES_REQUESTED** [review]
Thanks @AdrianMastronardi lgtm ex typo. ping on green.

**Review by datapythonista — APPROVED** [review]
lgtm, thanks @AdrianMastronardi 

@simonjayhawkins I think this is ready

**Review by simonjayhawkins — APPROVED** [review]

**Comment by simonjayhawkins** [comment]
Thanks @AdrianMastronardi

**Pull request merged**